docs/working-with-js-objects-and-values.md

```diff
@@ -202,7 +202,7 @@ manual](https://v2.ocaml.org/manual/objectexamples.html).
 We have already explored one approach for creating JavaScript object literals by
 using [`Js.t` values and the `mel.obj` extension](#using-js-t-objects).
 
-Melange additionally offers the `@mel.obj` attribute, which can be used in
+Melange additionally offers the `mel.obj` attribute, which can be used in
```

Comment:
Removed the `@` from attribute names that was added in [previous PR](https://github.com/melange-re/melange-re.github.io/pull/211#discussion_r1891097868).